.changeset/olive-cycles-count.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': patch
```

Comment:
These should be minor as it's changing the API.

Reply:
Fixed, thanks.